stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -1007,3 +1007,58 @@ function matmul3x3!(C::AbstractMatrix, tA, tB, A::AbstractMatrix, B::AbstractMat
     end # inbounds
     C
 end
+
+# Three-argument *
+*(A::AbstractMatrix, B::AbstractMatrix, x::AbstractVector) = _mat_mat_vec(A,B,x)
+
+*(α::Number, A::AbstractMatrix, x::AbstractVector) = _scalar_mat_vec(α,A,x)
+*(α::Number, A::AbstractMatrix, B::AbstractMatrix) = _scalar_mat_mat(α,A,B)
```

Comment:
And here (sorry!): alpha is multiplied from the left here, but from the right in `_scalar_mat_*`.